tmt/base.py

```diff
@@ -147,11 +147,13 @@ class FmfId(
     VALID_KEYS: ClassVar[list[str]] = ['url', 'ref', 'path', 'name']
 
     #: Keys that are present, might be set, but shall not be exported.
-    NONEXPORTABLE_KEYS: ClassVar[list[str]] = ['fmf_root', 'git_root', 'default_branch']
+    NONEXPORTABLE_KEYS: ClassVar[list[str]] = [
+        'fmf_root', 'anchor_path', 'git_root', 'default_branch']
 
     # Save context of the ID for later - there are places where it matters,
     # e.g. to not display `ref` under some conditions.
     fmf_root: Optional[Path] = None
+    anchor_path: Optional[Path] = None
```

Comment:
@falconizmi are these two changes needed? Why does `anchor_path` attribute sneak into an fmf ID?

Reply:
I mistook it as necessary thing for anchor_path getter. They are removed now.

Reply:
Seems to be solved.